app/controllers/users_controller.rb

```diff
@@ -303,26 +303,36 @@ def invited
 
     inviter = fetch_user_from_params(include_inactive: current_user.try(:staff?) || (current_user && SiteSetting.show_inactive_accounts))
     offset = params[:offset].to_i || 0
-    filter_by = params[:filter]
+    filter_by = params[:filter] || "redeemed"
 
     invites = if guardian.can_see_invite_details?(inviter) && filter_by == "pending"
       Invite.find_pending_invites_from(inviter, offset)
-    else
+    elsif guardian.can_see_invite_details?(inviter) && filter_by == "links"
+      Invite.find_links_invites_from(inviter, offset)
+    elsif filter_by == "redeemed"
       Invite.find_redeemed_invites_from(inviter, offset)
+    else
+      []
     end
 
     show_emails = guardian.can_see_invite_emails?(inviter)
-    if params[:search].present?
+    if params[:search].present? && invites.present?
       filter_sql = '(LOWER(users.username) LIKE :filter)'
       filter_sql = '(LOWER(invites.email) LIKE :filter) or (LOWER(users.username) LIKE :filter)' if show_emails
       invites = invites.where(filter_sql, filter: "%#{params[:search].downcase}%")
     end
 
-    render json: MultiJson.dump(InvitedSerializer.new(
-      OpenStruct.new(invite_list: invites.to_a, show_emails: show_emails, inviter: inviter),
-      scope: guardian,
-      root: false
-    ))
+    serialized = if filter_by == "links"
+      MultiJson.dump(invites: serialize_data(invites.to_a, InviteLinkSerializer), can_see_invite_details:  guardian.can_see_invite_details?(inviter))
```

Comment:
Are we returning 2 different formats of json depending on params?

Reply:
We need to use different serializer for links tab and pending/redeemed tabs. So if the param is `links` we use `InviteLinkSerializer` and for pending/redeemed we use `InvitedSerializer` serializer.

Reply:
structure wise if the format is different would we go with "links.json" for links and "invites.json" for pending & redeemed? its just a bit strange the the json format you get depends on the param, makes it harder for @oblakeerickson to document the API

Reply:
Understood. I'll differentiate the routes based on JSON response.

Reply:
This is now done.